SourceX/miniwin/misc_msg.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <SDL.h>
+
+namespace dvl {
+
+int translate_sdl_key(SDL_Keysym key);
+SDL_Keycode translate_dvl_key(int key);
```

Comment:
`#include "../../SourceS/sdl2_to_1_2_backports.h"` for SDL 1.2 compatibility

Reply:
Thanks :+1: